src/client/components/ui/Meilisearch/tests/MeilsearchCustomPagination.test.tsx

```diff
@@ -333,17 +333,16 @@ describe('MeilisearchCustomPagination', () => {
         }));
     });
 
-    it('affiche seulement 1 élément (le 1)', () => {
+    it('n affiche pas la pagination', () => {
```

Comment:
petit cadeau : ’ le vrai appostrophe courbe français, il ne pose pas de souci avec l'apostrophe qu'on utilise dans les it : pas besoin de le backstriper et on peut l'afficher plutôt que de mettre un espace :)

Reply:
a binder sur un shortcut